### DISCUSSION TITLE
Meeting: September 17th, 2024

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#153] [Announcements]
## Weekly meetings happen on Tuesdays

**Time:** 5pm GMT / 9am Pacific
**Zoom:** [https://zoom.us/j/975841675](https://zoom.us/j/975841675?pwd=SUh4MjRLaEFKNlI3RElpWTdhRDVVUT09), dial-in passcode: 763054

### Accessibility & Etiquette

* Participants must abide by our [Code-of-Conduct](https://github.com/OAI/OpenAPI-Specification/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).
* Meetings are recorded for future reference, and for those who are not able to attend in-person.
* We invited challenges to language or behaviour that is harmful or not inclusive during this meeting.
* We look forward to your participation, but please consider these acts of etiquette:
  * Remain on mute when not speaking to prevent interruptions.
  * Blur your background to reduce visual distractions.
  * Use the Zoom meeting "Raise Hand" feature to notify the group when you wish to speak.

| Blur My Background | Raise Hand |
|-|-|
| <img width="323" alt="Screenshot of Zoom UI showing the 'Stop Video' and 'Blur My Background' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/7e43dbbb-6529-46e6-8b04-4c1aa852d9dd"> | <img width="323" alt="Screenshot of Zoom UI showing the 'Reaction' and 'Raise Hand' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/f991722f-4651-40aa-9bc4-7e9a2a165a6a"> |

### Agenda Structure

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |
Wrap-up and seed the next agenda (5 mins) | SIG members | -- |

Whether attending or not, **anyone can comment on this issue prior to the meeting to suggest topics or to add comments on planned topics or proposals**.

**Comment by earth2marsh** [comment]
Picking up from last week, @handrews challenged us to put the diagram (link coming!) into practice by choosing one of the colored tool-spaces and seeing how its concepts can be applied. Hopefully we can cover 1-2 of those in this hour. If you want to collaborate, please use the `#moonwalk` channel in Slack to do so!

**Reply by handrews** [reply]
For anyone wondering what happened with the diagram (or, really, the whole series of various diagrams from the last few weeks): I started out just trying to tidy them up but realized they could be made much more clear.  There was also a request that I write up a bit of a guide to them, since not everyone heard the (frequently rambling) original explanations.

I've been hard at work on this all week but it has taken more time.  It won't be done before tomorrow's meeting but it's getting close- I've finished reworking the big diagram with the fields and am now reworking the "conceptual tooling architecture" diagram that gives the fields diagram context.  The write-ups are about halfway done.  I'm guessing maybe Wednesday before I put it all up.

**Comment by earth2marsh** [comment]
@darrelmiller may have some progress to report on some [proof-of-concept work](https://github.com/darrelmiller/oasbridge) around "internal representation"?

**Comment by earth2marsh** [comment]
We did, indeed, review Darrel's code explorations and Henry's diagram updates! [Video](https://zoom.us/rec/share/Vt1rFRSKSdWETsONFb-E2yLwm9mBPQjtide3m3C8u-QznqvkSiRLFpiydj3vWDUk.v_9Hu4QdfO-EifSS) of the meeting is now available.
